docs/advanced-usage/packaging-for-automated-evaluation.md

```diff
@@ -214,6 +214,20 @@ cases and our models.
 
 ## Appendix
 
+### Testing evaluator locally
+
+You can verify the evaluator Docker image by running it locally:
+
+```
+docker run --rm \
+  -e KOLENA_TEST_SUITE_NAME="my test suite" \
+  -e KOLENA_TEST_SUITE_VERSION=3 \
```

Comment:
Hmm, this one is a bit awkward, since we don't expect users to think about versions for the most part.

Reply:
Added information how user can get version information to mitigate this a bit.